src/domain/harbors/HarborsPageContainer.tsx

```diff
@@ -59,69 +62,107 @@ const HarborsContainer: React.FC = () => {
       Header: t('harbors.tableHeaders.places'),
       accessor: 'numberOfPlaces',
     },
+    {
+      Cell: ({ cell }) => (
+        <div>
+          <IconExclamationCircle
+            className={classNames(styles.icon, styles.nowrapper, {
+              [styles.disabled]: !cell.value,
+            })}
+          />
+        </div>
+      ),
+      Header: () => (
+        <div className={classNames(styles.icon, styles.nowrapper)}>
+          <IconExclamationCircle />
+        </div>
+      ),
+      accessor: 'id',
+    },
     {
       Cell: ({ cell }) => (
         <IconOutline disabled={!cell.value}>
-          <IconPole
-            className={classNames(styles.icon, styles.outline, {
+          <IconPlug
+            className={classNames(styles.icon, {
               [styles.disabled]: !cell.value,
             })}
```

Comment:
Wouldn't it be better if all the icons share the same styles, aka wrapped in a component, instead of having to pass the same `className` whenever we use it?
This approach defeats the "reusable components" practice in React!

Reply:
the Icon component in hds-react uses this signature